### DISCUSSION TITLE
how do i use this game

### DISCUSSION THREAD

**Discussion by theuwufish** [hdescottes/GdxGame#49] [Q&A] [answered]
so i downloaded this, how do i play it (i am new )

**Comment by hdescottes** [comment, marked as answer]
Hello, thanks for your interest in my work !
The game is still in development for now, even if it already has a lot of features. It has no story and only two maps so far.
But you can still play it by launching it from your favorite IDE :)

**Reply by theuwufish** [reply]
okay but which file do i open in the IDE (i use jgrasp)

**Reply by hdescottes** [reply]
I don't know much about jgrasp. What i can tell you is that in most IDE you will have to import the whole project in order to make it run.
Once this done, create a run application setting which will use the main class of the project (here it is [DesktopLauncher](https://github.com/hdescottes/GdxGame/blob/master/desktop/src/main/java/com/gdx/game/desktop/DesktopLauncher.java)).
